_docs/reference/release_notes.md

```diff
@@ -5,43 +5,126 @@ anchor: reference
-non-business critical infrastructure or devices.**
-
-Before the GA release, data formats used by StorageOS may change and there may
-not be an automated migration process. When upgrades break compatibility or
-require manual migration processes we will endeavour to make this clear in the
-release notes.
+# Release Notes
 
 We recommend always using "tagged" versions of StorageOS rather than "latest",
 and to perform upgrades only after reading the release notes.
 
+The latest tagged release is `1.0.0-rc1`, available from the
+[Docker Hub](https://hub.docker.com/r/storageos/node/) as
+`storageos/node:1.0.0-rc1`, or via the
+[Helm Chart](https://github.com/storageos/helm-chart)
+
+The latest CLI release is `1.0.0-rc1`, available from
+[Github](https://github.com/storageos/go-cli/releases)
+
 ## Upgrades
 
 Upgrades can be performed by restarting the StorageOS node container with the
-new version, or by performing `docker plugin upgrade` on Docker Managed Plugin
-installs.
+new version.
 
 Before a node is upgraded, applications local to the node that mount StorageOS
 volumes should be migrated to other cluster nodes or their data volumes will be
 unavailable while the StorageOS container restarts.
 
+See [maintenance]({%link _docs/operations/maintenance %}) for commands to help
+with online migration of volumes.
+
+See [maintenance]({%link _docs/operations/maintenance %}) for commands to help
+with online migration of volumes.
+
+The [StorageOS CLI](https://github.com/storageos/go-cli) should normally be
+updated when upgrading the cluster version.
+
 Where there are special considerations they are described below.
 
-During Beta upgrades may not always be possible as our focus will be on making
-bug fixes and new features available as soon as possible.  Once StorageOS
-reaches GA, more effort will be made to ensure that upgrades are seamless.
+### 0.10.x - 1.0.x
+
+Due to breaking changes between 0.10.x and 1.0.x upgrading is not possible.  We
+will now endeavour to allow upgrades between versions.
+
+If you are installing on a node that has had a previous version installed, make
+sure that the contents of `/var/lib/storageos` has been removed, and that you
+provision with a new cluster discovery token (if using).
+
+## 1.0.0-rc1
+
+The 1.0 release series is focussed on supporting enterprise workloads, with
+numerous new features and improvements to performance, stability and
+maintainability.
+
+### Breaking changes
 
-### 0.9.x -> 0.10.x
+- Several internal data structures were changed and are not compatible with
+  previous versions.  For this reason upgrades from version 0.10.x and earlier
+  are not supported.
+- The API endpoint `/v1/metrics` has been replaced with `/metrics` to conform
+  with Prometheus best practices.
+
+### New
+
+- Volume presentation has changed to use the SCSI subsystem via the
+  Linux SCSI Target driver. Previous versions used NBD where available, or FUSE
```

Comment:
I would add a reference to the compatibility page for LIO reference/os_support

Reply:
other than that I find the docs very precise and well explained.

Reply:
@Arau 

_docs/reference/os_support needs to be added to _guides/006-reference.md to show up in the left hand menu.

Reply:
@oicheryl  that was added on the lio_support branch as it will merge to this one I believe.